aria/cli/commands/service_templates.py

```diff
@@ -195,7 +195,9 @@ def create_archive(service_template_path, destination, logger):
     `destination` is the path of the output CSAR archive file
     """
     logger.info('Creating a CSAR archive')
-    csar.write(os.path.dirname(service_template_path), service_template_path, destination, logger)
+    if not destination.endswith('.csar'):
```

Comment:
maybe extract `.csar` suffix to a constant in csar.py? seems like its defined in several places now